app/models/miq_report_result.rb

```diff
@@ -6,7 +6,7 @@ class MiqReportResult < ApplicationRecord
   belongs_to :miq_task
   has_one    :binary_blob, :as => :resource, :dependent => :destroy
   has_many   :miq_report_result_details, :dependent => :delete_all
-  has_many   :html_details, -> { where "data_type = 'html'" }, :class_name => "MiqReportResultDetail", :foreign_key => "miq_report_result_id"
+  has_many   :html_details, -> { where("data_type = 'html'") }, :class_name => "MiqReportResultDetail", :foreign_key => "miq_report_result_id"
```

Comment:
Why isn't this just using the hash syntax?  This is already a problem, just don't get the reason for the string based `.where` here.